org.eclipse.xtext.util/src/org/eclipse/xtext/util/MergeableManifest2.java

```diff
@@ -316,25 +318,36 @@ private static String mergeIntoCommaSeparatedList(String bundle, String oldBundl
 		if (oldBundles.isEmpty())
 			return bundle;
 		String result = "";
```

Comment:
Even though it is not relevant here, but generally speaking a StringBuilder would be the better choice to create the result.

Reply:
Will change.